presto-docs/src/main/sphinx/presto-cpp.rst

```diff
@@ -45,8 +45,11 @@ Only specific connectors are supported in the Presto C++ evaluation engine.
 
 * Hive connector for reads and writes, including CTAS, are supported.
 
+* Hive connector supports Delta table access using Symlink table using Hive Connector in Prestissimo.
```

Comment:
```suggestion
* Hive connector supports Delta Lake table access using Symlink table using Hive Connector in Prestissimo.
```
Should it be "Delta table access", or "Delta Lake table access"? If the first, please ignore this comment.